app/Exceptions/Handler.php

```diff
@@ -60,17 +85,23 @@ protected function unauthenticated($request, AuthenticationException $exception)
             return response()->json(['error' => 'Unauthenticated.'], 401);
         }
         // For later if we have different users.
+
         $guard = array_get($exception->guards(), 0);
         switch ($guard) {
             case 'api':
                 $login = route('api.login');
                 break;
+            //TODO: merge
```

Comment:
What do you mean merge - is it that store isn't in yet?

Reply:
it's a notice for me to pay attention when i do the guard/route card. it could get squirrely at that point.

Reply:
:100: